components/KiboOrderItemDetails.vue

```diff
@@ -4,81 +4,116 @@
       <hr class="details-hr" />
     </div>
     <div class="order">
-      <div class="order-header-container order-item-box">
+      <div class="order-header-container order__item-box">
         <div class="order-header">
-          <div>{{ $t("Your Order") }} :</div>
-          <div>{{ $t("Order Date") }} :</div>
-          <div>{{ $t("Order Total") }}:</div>
+          <SfProperty :name="$t('Your Order:')" :value="orderNumber" />
```

Comment:
In case of SfProperty, we don't need to explicitly provide : for name.

Reply:
Yes, you are right but we have explicitly removed ":" from sf-property__name in the scss. 
 ```
   &::after {
      content: var(--property-name-content, "");
    }
&--without-suffix {
    --property-name-content: "";
  }
```
SfProperty used in so many places and we need to remove the css . We should use class "sf-property--without-suffix" where : is not required and default behavior os sfProperty should be applied by default. 
We can clean up the code later with proper testing.